### ISSUE TITLE
[BUG]onnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization: /onnxruntime_src/onnxruntime/core/providers/cpu/nn/batch_norm.h:52 onnxruntime::BatchNorm<T>::BatchNorm(const onnxruntime::OpKernelInfo&) [with T = float] !is_train_ was false. Training mode is not supported in this build.

### ISSUE THREAD

**Issue by zyzzu** [daquexian/onnx-simplifier#190]
**Describe the bug**
导出onnx时，training参数设置为TrainingMode.TRAINING会报错
**Model**
To reproduce the problem, please post download link of your model here, or send your model to daquexian566@gmail.com
